File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/GamesListPresenter.java
package bobrchess.of.by.belaruschess.presenter;

import bobrchess.of.by.belaruschess.view.activity.impl.GamesListActivity;

/**
 * Created by Igor on 12.04.2018.
 */

public interface GamesListPresenter extends BasePresenter {
    void loadGames();

    void loadGames(Integer count);

    void searchGames();

    void attachView(GamesListActivity activity);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/GamesListPresenter.kt
package bobrchess.of.by.belaruschess.presenter

import bobrchess.of.by.belaruschess.view.activity.impl.GamesListActivity

/**
 * Created by Igor on 12.04.2018.
 */

interface GamesListPresenter : BasePresenter {
    fun loadGames()

    fun loadGames(count: Int?)

    fun searchGames()

    fun attachView(activity: GamesListActivity)
}
